src/main/groovy/com/pszymczyk/consul/ConsulPorts.groovy

```diff
@@ -39,7 +42,8 @@ class ConsulPorts {
                     randomIfNotSet(dnsPort),
                     randomIfNotSet(serfLanPort),
                     randomIfNotSet(serfWanPort),
-                    randomIfNotSet(serverPort)
+                    randomIfNotSet(serverPort),
+                    -1
```

Comment:
so, grpcPort will always be -1?

Reply:
It can be improved to use random(Just not sure that I'll have time enough to do that).
I set it to `-1` to disable it. Because by Consul's defaults grpc port is always 8502 when running in 'dev' mode. And then some tests which run more that one consul instance fail.

Reply:
`randomIfNotSet(grpcPort)` would not work?